Sources/MapboxMaps/Style/Generated/Sources/GeoJSONSource.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
  */
 public struct GeoJSONSource: Source {
 
-    public let type: SourceType
+    public var type: SourceType
```

Comment:
This should still be a let, right?

Reply:
I changed it to a var since I had to add it to the protocol. This is so we could follow the same pattern that we did for layer.

Reply:
Replied in a new comment.